homeassistant/components/backup/config.py

```diff
@@ -51,6 +52,7 @@ class BackupConfigData:
     """Represent loaded backup config data."""
 
     agents: dict[str, AgentConfig]
+    automatic_backups_configured: bool
```

Comment:
maybe there should be a comment here explaining this is used by frontend and ignored by core

Reply:
Sounds good.